app/services/boarding_service.rb

```diff
@@ -56,7 +56,8 @@ def add_tester(email, first_name, last_name)
     end
 
     if app.apple_id.length > 0
-      Rails.logger.info "Addding tester to application to groups: #{tester_groups.to_s}"
+      groups = tester_groups || "External Testers"
```

Comment:
`tester_groups` is an array, but when we print out the message, we're using the string value. If it's nil, I'm setting `groups` to a string. I could make it less confusing my setting it to an array instead.